.github/workflows/code_coverage.yml

```diff
@@ -1,14 +1,9 @@
 name: Code Coverage
 
 on:
-  push:
-    branches:
-      - master
-    paths-ignore:
-    - 'docs/**'
-    - '.github/workflows/**'
-    - '!.github/workflows/code_coverage.yml'
-  pull_request:
+  schedule:
+  - cron:  '0 0 * * 0'
```

Comment:
I think we should leave the code coverage workflow on pull requests and master instead of moving it to a cron job, that way we can see differences on pull requests.

Reply:
Got it, reverted.